python/selfie-lib/selfie_lib/Literals.py

```diff
@@ -41,6 +42,36 @@ def parse(self, string: str, language: Language) -> T:
 PADDING_SIZE = len(str(MAX_RAW_NUMBER)) - 1
 
 
+def _encode_underscores(
+    buffer: io.StringIO, value: int, language: Language
```

Comment:
Make this a private method of `LiteralInt`, unless there's a good reason not to.

Reply:
Fixed this, originally just made it match the Kotlin code.